src/services/discord/commands/info/infoCmds.js

```diff
@@ -0,0 +1,115 @@
+const { MessageEmbed } = require('discord.js')
+const { version, colors, channels } = require('../../info')
+
+module.exports = {
```

Comment:
Clean export. Love it. Every property operates just like the others so a reader knows what to expect.